src/theme.module.css

```diff
@@ -13,8 +13,10 @@
     /* Profile page */
 
     --pink-color: #e14eca;
-    --succses-color: #00f2c3;
+    --success-color: #00f2c3;
```

Comment:
Have you checked if the "succses-color" is used anywhere else and if it needs to be updated there too?

Reply:
yes, I did, no use for it yet, only "green-color" has been used (witch is the same one)